iphone/Maps/UI/Search/Filters/MWMSearchHotelsFilterViewController.mm

```diff
@@ -56,18 +91,14 @@ void configButton(UIButton * button, NSString * primaryText, NSString * secondar
 }
 }  // namespace
 
-@interface MWMSearchHotelsFilterViewController ()
-
-@property(nonatomic) IBOutletCollection(UIButton) NSArray * ratings;
-
-@property(weak, nonatomic) IBOutlet UIButton * ratingAny;
-@property(weak, nonatomic) IBOutlet UIButton * rating7;
-@property(weak, nonatomic) IBOutlet UIButton * rating8;
-@property(weak, nonatomic) IBOutlet UIButton * rating9;
+@interface MWMSearchHotelsFilterViewController () <UICollectionViewDelegate, UICollectionViewDataSource>
+{
+  std::vector<ftypes::IsHotelChecker::Type> m_selectedTypes;
```

Comment:
Why not to replace vector by set? I'm not talking about performance, from my point of view insertion and removal will be much simpler.